### ISSUE TITLE
"Illegal instruction (core dumped)" error, when I tried to do the "installation from source".

### ISSUE THREAD

**Issue by hideaki-j** [informagi/REL#53]
I did an [installation from source](https://github.com/informagi/REL#installation-from-source) but the "Illegal instruction (core dumped)" error occurred when I tried to run it.

This error occurs in PyTorch in Flair. If you use the latest version of PyTorch (1.6.0) and an old CPU that does not support AVX, then this error occurs. You can avoid this error by using PyTorch (1.5.1). For more details, see [here](https://github.com/pytorch/pytorch/issues/43551). 

**Do you think it is better to specify the version of the PyTorch should be installed in the REL's readme document?**

## Details

If you call [```find_mentions```](https://github.com/informagi/REL/blob/master/REL/server.py#L120) like:

```
...
mentions_dataset, total_ment = self.mention_detection.find_mentions(processed, tagger_ner)
```
then you get
```
Illegal instruction (core dumped)
```
when you are using
- the latest version of PyTorch (1.6.0) and 
- CPU that does not support AVX

## Solution
Uninstall the PyTorch 1.6.0 and install PyTorch 1.5.1

**Comment by arjenpdevries** [comment]
Does PyTorch 1.6.0 not have a flag to turn off AVX?

**Comment by KDercksen** [comment]
If you build PyTorch from source, it should work just fine. The binary uses AVX by default I believe (I ran into issues with this on Gem as well). As far as I know, there is no way to turn off AVX/AVX2 using the prebuilt binaries (i.e. the library you get by installing through pip or conda).

**Comment by arjenpdevries** [comment]
I see: https://github.com/pytorch/pytorch/issues/43300
Wait till 1.7...

**Comment by KDercksen** [comment]
https://pytorch.org/blog/pytorch-1.7-released/ we might be in luck! :D

**Comment by arjenpdevries** [comment]
So maybe a quick warning in the how-to-install and then hope it's fixed for 1.7!
